crates/holochain_sqlite/src/stats.rs

```diff
@@ -0,0 +1,12 @@
+use crate::error::DatabaseError;
+use rusqlite::Transaction;
+
+pub fn get_size_on_disk(txn: Transaction) -> Result<usize, DatabaseError> {
+    txn.query_row("select sum(pgsize) from dbstat", (), |r| r.get(0))
```

Comment:
I changed my mind... Obviously it can load pages out of the WAL file when capturing read locks, so it probably can include those pages in the dbstat virtual table.